core/shared/src/main/scala/zio/prelude/AssociativeFlatten.scala

```diff
@@ -72,17 +72,17 @@ object AssociativeFlatten {
   /**
    * The `AssociativeFlatten` instance for `Const`.
    */
-  implicit def ConstAssociativeFlatten[A]: AssociativeFlatten[({ type ConstA[+B] = Const[A, B] })#ConstA] =
-    new AssociativeFlatten[({ type ConstA[+B] = Const[A, B] })#ConstA] {
+  implicit def ConstAssociativeFlatten[A]: AssociativeFlatten[Const[A, +*]] =
+    new AssociativeFlatten[Const[A, +*]] {
```

Comment:
We may want to choose to switch to kind-projector syntax, but I'd prefer we do it in a dedicated PR.
There also may be some issues related to this new syntax -- there were reasons to not use it in the beginning after all...